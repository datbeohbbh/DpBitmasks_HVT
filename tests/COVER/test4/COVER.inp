1000 5141 8980
1900 1952
6235 6297
1312 1319
3390 3425
9924 9989
1 67
2442 5309
9584 9612
5639 5706
4849 4853
3509 3552
6673 6734
6547 6592
6208 6245
5791 5846
5345 5354
9756 9800
2219 2237
9896 9954
3933 4025
419 424
8946 8985
564 629
2160 2189
7246 7278
736 794
5934 5970
3022 3034
6587 6645
6401 6444
1 48
9856 9865
5685 5712
9506 9508
268 317
2210 2235
7493 7506
9765 9798
703 768
8252 8303
1134 1157
7339 7352
9844 9907
8187 8233
7171 7201
2703 2754
9930 9934
3070 3101
1632 1688
7907 7940
6508 6568
220 263
1184 1205
1046 1101
5439 5456
9669 9715
4864 4882
3430 3488
6204 6235
3180 3187
1119 1148
6489 6552
5219 5222
8391 8453
1780 1808
6899 6949
764 816
565 582
5518 5571
71 78
9173 9218
2220 2269
7831 7838
1540 1603
1812 1879
189 196
6446 6473
6949 6977
615 677
1214 1222
8185 8243
9043 9087
2136 2406
5924 5963
4810 4831
7600 7636
8723 8783
5380 5426
6007 6047
6358 6384
2835 2857
2770 2836
2189 2219
8201 8249
9517 9584
504 564
2710 2726
6817 6830
6746 6775
7808 7826
3101 3121
7959 7990
2307 2359
9287 9323
9271 9317
2152 2156
9752 9811
6444 6489
755 778
36 88
7788 7846
9830 9867
535 548
5234 5281
5354 5380
5912 5978
500 541
9965 10000
2672 2699
6911 6940
1229 1234
1885 1908
299 305
6940 9618
5293 5345
9887 9914
8022 8088
1952 1998
8145 8167
2515 2545
5613 5639
3479 3500
8091 8136
5313 5376
2569 2618
2726 2730
4176 4263
9868 9914
2932 2981
348 367
7812 7853
8249 8276
6708 6746
8804 8833
8935 8961
6691 6750
5687 5740
320 372
85 140
8833 8834
7853 7885
836 843
2572 2627
6092 6097
2533 2552
438 497
9846 9889
8791 8826
1089 1154
5673 5687
1935 1941
430 438
5869 5924
305 348
222 268
6958 6983
479 537
8885 8935
7934 8779
9612 9638
9905 9965
7598 7603
9161 9164
3500 3557
923 1666
9700 9710
3718 3781
1674 1735
8088 8100
7970 7993
3263 3302
126 175
7552 7564
6126 6141
9640 9703
9985 10000
7826 7875
5276 5293
6043 6103
6552 6587
1457 1468
3024 3036
6137 6151
8725 8771
3146 3212
409 425
9418 9463
7603 7621
6475 6492
2452 2498
1157 1159
210 253
5285 5286
3202 3259
2569 2596
2775 2838
4646 4653
5414 5470
9280 9340
8324 8379
562 573
9819 9856
5743 5791
833 899
7929 7955
2012 2076
8429 8471
3557 3594
5202 5219
1365 1415
9903 9930
5482 5502
5502 5567
400 466
9811 9830
2121 2130
9044 9072
2031 2056
8518 8576
1585 1616
6064 6657
2240 2282
4688 4707
3359 3393
8018 8034
924 954
8670 8693
9989 10000
1009 1055
9638 9698
768 830
185 222
208 210
1184 1192
22 79
882 5222
8779 8804
1159 1212
3302 3367
9323 9364
2491 2515
6128 6137
424 459
3121 3142
1 16
469 500
3636 3644
5914 5927
8619 8681
5554 5617
3034 3040
1962 1990
8681 8725
8276 8319
3393 3431
9633 9640
383 384
5422 5488
3364 3430
645 709
2848 2906
4707 4725
4831 4864
6784 6794
5645 5717
5567 5570
9670 9695
384 438
9914 9931
2757 2783
9865 9887
7894 7948
5640 5643
5643 5651
2545 2569
2482 2491
851 853
2144 2181
194 220
2271 2307
3008 3053
7327 7377
536 565
736 755
8100 8145
5577 5581
2809 2835
145 194
1450 1476
3367 3390
425 469
9744 9765
6871 6899
1979 2012
2056 2087
8403 8429
7462 7508
6851 6899
9252 9279
372 383
7838 7896
9805 9807
1769 1796
9822 9868
109 157
420 430
7846 7894
1281 1318
4388 4408
120 186
1752 1799
7552 7600
932 993
8233 8247
459 515
6685 6691
1921 1938
9715 9752
2908 2954
3187 3221
6297 6300
8374 8403
5924 5957
1707 1752
2042 2058
988 1046
8372 8374
8985 9043
2507 2569
5222 5285
9763 9768
2390 2450
1589 1648
794 833
1666 1719
8319 8345
6402 6435
7671 7737
5651 5663
3086 3146
2887 2888
2235 2272
5571 5577
784 794
466 504
573 577
1616 1632
805 836
117 138
635 701
907 949
4774 4778
5632 5640
1555 1562
8154 8214
6065 6126
607 647
382 400
1476 1523
8924 8946
9007 9043
1367 1394
7672 7693
8214 8250
8653 8670
5050 5052
571 605
2317 2343
6250 6267
5160 5165
3142 3202
1875 1890
6141 6169
6225 6245
5015 5050
9364 9370
175 238
8771 8820
2041 2050
1415 1450
3880 3931
5388 5414
6589 6622
497 536
870 880
1362 1405
6492 6552
7993 8034
381 408
54 120
7506 7552
1716 1753
2645 2674
9934 9984
993 1004
5978 5997
2282 2317
5999 6043
5380 5381
430 459
6171 6225
2618 2626
9149 9197
409 419
4734 4774
8576 8588
8834 8841
1240 1295
7541 7551
6118 6171
4882 4913
962 1024
7030 7092
1955 1971
8167 8187
373 402
5712 5729
6734 6773
4408 4454
9087 9118
843 907
347 364
3040 3086
9807 9808
6350 6381
7737 7802
524 540
2739 2775
9808 9844
7948 7959
2343 2405
8592 8653
1974 2019
1648 1707
7955 7956
9618 9653
1494 1555
8368 8391
6103 6128
317 381
5165 5202
1154 1184
459 464
8286 8324
1845 1854
366 381
4347 4388
7940 7959
9257 9271
8588 8592
575 610
2380 2390
6267 6313
6523 6547
548 575
3283 3309
6830 6871
2642 2655
7747 7788
5414 5422
2836 2887
91 126
5886 5934
2405 2442
6348 6402
1562 1589
5793 5831
5488 5530
1192 1214
8243 8286
9279 9287
7659 7722
2981 4688
4100 4109
9085 9149
4579 4605
9043 9085
3036 3070
2480 2526
6441 6473
4809 5873
6097 6118
577 615
6381 6446
6899 7327
8693 8723
5012 5015
2359 2380
692 756
3352 3364
9798 9826
367 392
5710 5738
6473 6475
6674 6708
3173 3180
9914 9924
6634 6655
6622 6661
949 988
3328 3345
5617 5645
2343 2393
5530 5547
392 420
438 459
3431 3495
2970 3024
7787 7812
3553 3572
677 791
1719 1780
5510 5511
1112 1173
3683 3856
305 341
9218 9241
5970 6034
2417 2482
1316 1367
5740 5797
8964 8992
67 91
7885 7929
6795 6817
9907 9935
7693 7715
7352 7402
1890 1900
1173 1240
5663 5685
2270 2271
2028 2092
2473 2480
270 304
9698 9763
1 36
5896 5914
2272 2290
6169 6208
7168 7171
3604 3636
2237 2286
4897 4913
8521 8563
2626 2686
9710 9756
1799 1812
8900 8908
5581 5626
9626 9669
3114 3172
364 370
2050 2078
4980 5204
2891 2932
9197 9257
6586 6634
2396 2417
1915 1974
381 382
389 391
5738 5759
5729 5793
5052 5087
6839 6862
5470 5483
6592 6634
6674 8154
1076 1089
8034 8061
5875 5912
7508 7541
5472 5505
1854 1921
1 48
1523 1578
5281 5339
304 305
6091 6132
6842 6858
1274 1281
8961 9007
263 278
4955 5012
7241 7246
48 1674
2674 2703
7722 7728
8453 8518
540 571
2627 2645
9653 9664
9508 9573
2181 2240
1468 1470
8820 8842
2888 2929
909 927
4811 4818
8034 8083
1055 1090
5393 5439
2393 2396
1222 1262
4263 4294
4454 4465
1024 1076
1796 1849
3488 3549
791 851
1992 2031
6047 6065
9768 9822
2606 2642
1990 2042
5998 5999
2269 2270
3644 3677
4804 4809
1148 1184
262 273
16 63
681 689
846 851
2838 2884
391 409
2450 2456
7875 7934
2058 2121
347 366
2076 2136
4605 4646
2941 2980
9957 10000
5852 5913
3495 3509
3309 3359
1318 1348
6151 6204
4109 4176
7092 7109
6862 6892
9826 9846
9760 9816
6568 6589
5483 5518
5850 5896
6794 6839
6023 6064
2552 2559
1643 1709
2121 2166
5547 5613
5358 5388
6067 6091
1536 1540
4881 4897
9340 9392
5958 5998
6245 6312
8471 8516
327 382
3931 3933
6983 7030
5963 6023
2290 2404
6384 6401
8345 8368
7621 7635
7564 7593
3781 3809
5381 5414
6655 6674
4465 4495
1941 1992
5237 5276
2406 2430
6435 6441
402 430
3809 3844
6810 6842
954 1009
689 736
5953 5958
9664 9700
2404 2438
5507 5510
4025 4035
7896 7899
87 109
7802 7831
5957 6000
3172 3173
816 882
4778 4810
652 692
6552 6586
2596 2606
9456 9517
1855 1915
2980 3022
1971 1979
6766 6774
9143 9173
899 923
3425 3479
7432 7493
464 479
5222 5237
9118 9143
6300 6348
5759 5770
9745 9805
2884 2891
7109 7168
830 832
138 185
5286 5313
5204 5234
7956 8022
9241 9280
6067 6092
2783 2809
9113 9161
9984 10000
4653 4679
6977 9670
8303 8345
3552 3589
1808 1860
8143 8201
2430 2477
4570 4579
2699 2710
5913 5924
832 877
5831 5852
6473 6508
2087 2121
4495 4523
6750 6766
5456 5472
3594 3718
7899 7907
6245 6250
3856 3880
6657 6673
4745 4752
273 326
6661 6689
1938 1955
5706 5710
7635 7659
2712 2739
4913 4944
2906 2932
2932 4745
537 562
3259 3283
7636 7672
582 652
3221 3263
9800 9819
8783 8791
2655 2712
9816 9841
5376 5393
619 635
4853 4875
5511 5554
6665 6685
6446 6498
1795 1855
1753 1795
9841 9903
2857 2908
9931 9948
5927 5953
3572 3604
6858 6911
341 347
9463 9506
3844 7808
79 85
9867 9877
8563 8619
3261 3302
78 117
1302 1365
1101 1112
2477 2503
877 924
157 208
2286 2343
5873 5875
9317 9626
6689 6754
9877 9896
1041 1089
2664 2672
6773 6784
253 303
3648 3683
1004 1027
6000 6067
8119 8143
186 219
7426 7462
8850 8885
8992 9044
927 962
5797 5850
7201 7206
7402 7432
6034 6067
1415 1479
9091 9113
701 708
4818 4849
9935 10000
9695 9760
5318 5380
303 320
7377 7426
3589 3648
1941 1962
4913 4955
8842 8878
7715 7786
6780 6795
5505 5507
1 54
610 619
5997 6007
1479 1536
880 909
278 299
5826 5869
1735 1845
4744 4804
9164 9202
778 784
2097 2160
6952 6958
9948 9985
5087 5116
63 71
1998 3008
459 524
1319 1362
5770 5826
851 932
48 87
1287 1316
1262 1302
2498 2533
9686 9745
1603 1643
8247 9640
4944 4980
2748 2757
5426 5482
794 846
4725 4734
8110 8119
196 262
2729 2748
9954 9957
9889 9905
4294 4342
8841 8858
1348 1415
9072 9091
7551 7552
2156 2220
767 805
1089 1119
8061 8065
5626 5673
8858 8924
2526 2572
1470 1494
1205 1229
1301 1312
5570 5632
1709 1769
8826 8850
756 764
647 703
853 870
2559 2595
1394 1457
3345 3352
2019 2041
6132 6402
7278 7339
5717 5743
6402 6446
8065 8110
1234 1287
2078 2144
7786 7787
8136 8185
709 767
8250 8252
2166 2192
708 736
6892 6952
7959 7970
5309 5318
88 145
4875 4881
5116 5160
1879 1941
1849 1875
1578 1585
2130 2152
6774 6780
238 281
2456 2473
2754 2792
7206 7241
6331 6350
3212 3261
326 347
6754 6766
2595 2664
2954 2970
8083 8091
219 270
281 327
2438 2452
6634 6665
6795 6851
1688 1716
2092 2097
7593 7598
4342 4347
9392 9456
2730 2770
3549 3553
1090 1134
6498 6523
541 607
1027 1041
1405 2028
3708 7671
1295 1301
605 645
2929 2941
6312 6331
8878 8900
5339 5358
7990 8018
6766 6810
4523 4570
2503 2507
3302 3328
9703 9744
5846 5886
6775 6795
4679 4744
629 681
4035 4100
1908 1935
1860 1885
382 389
6313 6358
1212 1274
9573 9633
2686 2729
140 189
4752 4811
8516 8521
6645 6674
3677 3708
8908 8964
9202 9252
1 22
2192 2210
2792 2848
408 409
8345 8372
9640 9686
515 535
3053 3114
370 373
9370 9418
7728 7747
