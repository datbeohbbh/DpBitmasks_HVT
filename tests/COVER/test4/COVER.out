2
738
794
